lib/queryhelpers.js

```diff
@@ -135,7 +135,7 @@ exports.applyPaths = function applyPaths(fields, schema) {
       if (!isDefiningProjection(field)) {
         continue;
       }
-      exclude = field === 0;
+      exclude = field === 0 || field === false;
```

Comment:
I think it would be better for this to be `exclude = !field`. Better to use [falsy](https://masteringjs.io/tutorials/fundamentals/falsy) rather than hard coding `0` and `false` as the allowed values while leaving `0n` as equivalent to `true`.

Reply:
Sounds reasonable.